packages/common/client-utils/package.json

```diff
@@ -180,21 +180,39 @@
 				"build:test:jest",
 				"build:test:types"
 			],
-			"build:api-report:browser:current": [
+			"build:api-reports:browser:current": [
 				"build:exports:browser"
 			],
-			"build:api-report:browser:legacy": [
+			"build:api-reports:browser:legacy": [
 				"build:exports:browser"
 			],
-			"build:api-report:node:current": [
-				"build:api-report:browser:current"
+			"build:api-reports:node:current": [
+				"build:api-reports:browser:current"
 			],
-			"build:api-report:node:legacy": [
-				"build:api-report:browser:legacy"
+			"build:api-reports:node:legacy": [
+				"build:api-reports:browser:legacy"
+			],
+			"build:exports:browser": [
+				"build:esnext"
+			],
+			"build:exports:node": [
+				"build:esnext"
+			],
+			"check:exports:cjs:indexBrowser": [
+				"tsc"
+			],
+			"check:exports:cjs:indexNode": [
+				"tsc"
+			],
+			"check:exports:esm:indexBrowser": [
+				"build:esnext"
```

Comment:
Yeah. Think it is uncommon and not likely to be common. If it replicates, then sure we can relocate. As it is all very handcrafted.
It would be nice if we could quickly infer some of these dependencies instead of having them have to be explicit. Maybe the compromise is to have it as policy and have a fixer. So, it becomes a "cache" of the inferred dependencies.